# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iptraf/cross-cc.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iptraf-3.0.0/support/Makefile.vanilla	2007-08-15 13:21:29.000000000 +0200
+++ iptraf-3.0.0/support/Makefile	2007-08-15 13:21:49.000000000 +0200
@@ -7,12 +7,12 @@
 
 libtextbox.a: $(OBJS)
 	rm -rf libtextbox.a
-	ar cq libtextbox.a $(OBJS)
-	ranlib libtextbox.a
+	$(AR) cq libtextbox.a $(OBJS)
+	$(RANLIB) libtextbox.a
 #	gcc -shared -o libtextbox.so $(OBJS)
 
 %.o: %.c *.h
-	gcc -O2 -g -Wall -fPIC $(INCLUDEDIR) -c -o $*.o $<
+	$(CC) -O2 -g -Wall -fPIC $(INCLUDEDIR) -c -o $*.o $<
 
 clean:
 	rm -rf *.o *~ libtextbox.a libtextbox.so
